cdmtaskservice/mongo.py

```diff
@@ -219,6 +220,32 @@ def _doc_to_job(self, doc: dict[str, Any], as_admin: bool = False
         doc = self._clean_doc(doc)
         return models.AdminJobDetails(**doc) if as_admin else models.Job(**doc)
 
+    async def list_jobs(self, user: str | None = None, limit: int = 1000
+    ) -> list[models.JobPreview]:
+        """
+        List jobs.
+        
+        user - filter jobs by user.
+        limit - the maximum number of jobs to return.
```

Comment:
Could you remind me why we are setting a `limit`? 1000 is a pretty low count in general. Are we worrying about the payload size here?

Reply:
There has to be some maximum amount of data a user can request, and 1000 is a) plenty, and b) allows for 100us per document and will still return in 100ms

Reply:
As a point of reference, the workspace limit is 10k objects and that returns in ~2s, although it's more complicated than that. I want to keep it sub human reaction times